### ISSUE TITLE
pinax-notifications 4.x won't install templates and locales

### ISSUE THREAD

**Issue by cezio** [pinax/pinax-notifications#73]
Notifications won't install templates data (``templates/pinax/notifications/*``) in recent version, when installed through pypi. I believe this is caused by:
https://github.com/pinax/pinax-notifications/commit/07076ff71eba685b55deb82ccb88818fd64ada7c#diff-2eeaed663bd0d25b7e608891384b7298 :

4.0:
```
cezio@gburek [tmp $]>mkvirtualenv blank
New python executable in /home/cezio/.virtualenvs/blank/bin/python2.7
Also creating executable in /home/cezio/.virtualenvs/blank/bin/python
Installing setuptools, pip, wheel...done.
virtualenvwrapper.user_scripts creating /home/cezio/.virtualenvs/blank/bin/predeactivate
virtualenvwrapper.user_scripts creating /home/cezio/.virtualenvs/blank/bin/postdeactivate
virtualenvwrapper.user_scripts creating /home/cezio/.virtualenvs/blank/bin/preactivate
virtualenvwrapper.user_scripts creating /home/cezio/.virtualenvs/blank/bin/postactivate
virtualenvwrapper.user_scripts creating /home/cezio/.virtualenvs/blank/bin/get_env_details
 
cezio@gburek [tmp $]>pip install pinax-notifications
Collecting pinax-notifications
  Using cached pinax_notifications-4.0.0-py2.py3-none-any.whl
Collecting django-appconf>=1.0.1 (from pinax-notifications)
  Using cached django_appconf-1.0.2-py2.py3-none-any.whl
Installing collected packages: django-appconf, pinax-notifications
Successfully installed django-appconf-1.0.2 pinax-notifications-4.0.0

cezio@gburek [tmp $]>find /home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/ -type d
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/migrations
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/management
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/management/commands
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/backends
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/tests
```
3.x:
```
cezio@gburek [tmp $]>pip install 'pinax-notifications<4.0'
Collecting pinax-notifications<4.0
  Using cached pinax_notifications-3.0.1-py2.py3-none-any.whl
Requirement already satisfied: django-appconf>=1.0.1 in /home/cezio/.virtualenvs/blank/lib/python2.7/site-packages (from pinax-notifications<4.0)
Installing collected packages: pinax-notifications
  Found existing installation: pinax-notifications 4.0.0
    Uninstalling pinax-notifications-4.0.0:
      Successfully uninstalled pinax-notifications-4.0.0
Successfully installed pinax-notifications-3.0.1
 
cezio@gburek [tmp $]>find /home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/ -type d
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/migrations
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/templates
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/templates/pinax
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/templates/pinax/notifications
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/es_AR
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/es_AR/LC_MESSAGES
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/ru
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/ru/LC_MESSAGES
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/fr
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/locale/fr/LC_MESSAGES
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/management
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/management/commands
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/backends
/home/cezio/.virtualenvs/blank/lib/python2.7/site-packages/pinax/notifications/tests
```

**Comment by pavanv** [comment]
I also hit this today after upgrading to Pinax 4.0.0.

@cezio how did you fix your way through it?

**Comment by cezio** [comment]
i've forced pre-4.x version. Permanent fix would be to have ``package_data`` value restored in ``setup.py``.

**Comment by surfer190** [comment]
Yeah quite disappointing that this issue is not fixed and managed to occur in the first place.
How do we write a test and fix this?

**Comment by paltman** [comment]
Sorry, folks.  I missed this on my review of the merge.  We'll get in a fix and re-release before the Pinax 18.01 distribution, hopefully.

**Comment by paltman** [comment]
fixed in 42b31b2
